pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -291,6 +291,21 @@ public static class QueryOptionKey {
 
         public static final String ORDER_BY_ALGORITHM = "orderByAlgorithm";
 
+        // Number of rows limit to select in leaf stage.
+        // It has to be greater than 0.
+        // If it is invalid number format, or the number is smaller than or equal to zero, we use
+        // DEFAULT_LEAF_NODE_LIMIT.
+        public static final String MS_LEAF_LIMIT = "multiStageLeafLimit";
+
+        // Override numGroupLimit used by v1 engine.
+        public static final String NUM_GROUP_LIMIT = "numGroupLimit";
+
+        // Override maxInitialResultCap used by v1 engine.
+        public static final String MAX_INITIAL_RESULT_HOLDER_CAPACITY = "maxInitialResultCap";
+
+        // Override groupByTrimThreshold used by v1 engine.
+        public static final String GROUP_BY_TRIM_THRESHOLD = "groupByTrimThreshold";
```

Comment:
Suggest keeping the naming convention:
```suggestion
        public static final String MULTI_STAGE_LEAF_LIMIT = "multiStageLeafLimit";
        public static final String NUM_GROUPS_LIMIT = "numGroupsLimit";
        public static final String MAX_INITIAL_RESULT_HOLDER_CAPACITY = "maxInitialResultHolderCapacity";
        public static final String GROUP_TRIM_THRESHOLD = "groupTrimThreshold";
```

Reply:
Done